cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pod
+from cfme.containers.route import Route, list_tbl as list_tbl_route
+from cfme.containers.project import Project, list_tbl as list_tbl_proj
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9911 # CMP-9877 # CMP-9867
+
+
```

Comment:
As you are navigating to the page showing _all_ pods, I think you either want to
a) use the ```has_no_container_providers``` fixture to remove all providers before you run the test, probably meaning you need to define some other fixtures, to get the dependency right
b) Navigate to Pods (through) the provider, so tat you only get pods that are related to the provider you are using.